lib/srv/ctx.go

```diff
@@ -722,6 +729,13 @@ func (c *ServerContext) ExecCommand() (*ExecCommand, error) {
 		requestType = c.request.Type
 	}
 
+	hostname := c.srv.GetInfo().GetHostname()
```

Comment:
Include a comment in this block like the ones above explaining you are getting things ready for utmp.